Project 1/styles.css

```diff
@@ -0,0 +1,56 @@
+body {
```

Comment:
Consider adding a `color` and base `font-family` rule to your `body` element here. This would also save you repeating the `color` styles on individual elements.